frame/vamm/src/lib.rs

```diff
@@ -529,9 +561,60 @@ pub mod pallet {
 			}
 		}
 
-		#[allow(unused_variables)]
-		fn get_twap(vamm_id: &VammIdOf<T>) -> Result<DecimalOf<T>, DispatchError> {
-			todo!()
+		/// Returns the time weighted average price of the desired asset.
+		///
+		/// # Overview
+		/// In order for the caller to know which is the time weighted average
+		/// price of the desired asset, it has to request it to the Vamm Pallet.
+		/// The pallet will query the runtime storage and return the desired
+		/// twap.
+		///
+		/// ![](https://www.plantuml.com/plantuml/svg/FSqz3i8m343XdLF01UgTgH8IrwXSrsqYnKxa7tfzAWQcfszwimTQfBJReogrt3YjtKl4y2U0uJaTDKgkwMpKDLXZeYxmwZAwuzhuNO7-07OgRB0R2iC7HM2hU5nos5CfQjVbu5ZYn36DXlfxpwpRrIy0)
+		///
+		/// ## Parameters
+		///  - [`vamm_id`](Config::VammId): The ID of the desired vamm to query.
+		///  - [`asset_type`](composable_traits::vamm::AssetType): The desired
+		///  asset type to get info about.
+		///
+		/// ## Returns
+		/// The twap for the specified asset.
+		///
+		/// ## Assumptions or Requirements
+		/// * The requested [`VammId`](Config::VammId) must exists
```

Comment:
exists -> exist. You can add this to another PR tho so you don't have to redo this one

Reply:
Thanks! :)